### PR TITLE
Job offer be

### PR DESCRIPTION
**Pull request by Yulllia**
- add offer saving in database
- get and updating status

### CODE REVIEW

File: src/modules/inviteTalent/inviteTalent.service.ts [hunk 1]
```diff
@@ -19,9 +19,22 @@ export class InviteTalentService {
       newMessage.profileId = InviteTalentDto.profileId;
       newMessage.jobTitle = InviteTalentDto.jobTitle;
       const invitation = await this.inviteTalentRepository.save(newMessage);
+      console.log(invitation);
```
Comment (lyudad) [inline src/modules/inviteTalent/inviteTalent.service.ts]: delete

File: src/modules/inviteTalent/inviteTalent.service.ts [hunk 2]
```diff
@@ -19,9 +19,22 @@ export class InviteTalentService {
...
       return invitation;
     } catch (e) {
       console.log(e);
     }
   }
+
+  async getInviteMessageByProfile(userId: number) {
```
Comment (lyudad) [inline src/modules/inviteTalent/inviteTalent.service.ts]: return types

File: src/modules/inviteTalent/inviteTalent.service.ts (hunk 1 above)
Comment (Yulllia) [reply src/modules/inviteTalent/inviteTalent.service.ts]: fixed

File: src/modules/inviteTalent/inviteTalent.service.ts (hunk 2 above)
Comment (Yulllia) [reply src/modules/inviteTalent/inviteTalent.service.ts]: fixed

**Pull request merged**